target-chains/cosmwasm/tools/src/deployer/injective.ts

```diff
@@ -169,6 +170,38 @@ export class InjectiveDeployer implements Deployer {
     }
   }
 
+  async updateAdmin(newAdmin: string, contract: string): Promise<void> {
```

Comment:
I suggest documenting that this function assumes that this deployer is the current admin for the contract, and this will fail if this condition isn't satisfied. (also, does it fail with a reasonable error message?)

Reply:
it doesn't fail with a reasonable error message. but that is the case for most of the places.
i am gonna raise another pr for error handling.